overlord/configstate/configcore/netplan.go

```diff
@@ -49,22 +56,44 @@ func init() {
 	config.RegisterExternalConfig("core", "system.network.netplan", getNetplanFromSystem)
 }
 
-func validateNetplanSettings(tr config.Conf) error {
-	// XXX: validate somehow once we support writing?
-	return nil
+type connectivityCheckStore interface {
+	ConnectivityCheck() (map[string]bool, error)
 }
 
-func handleNetplanConfiguration(tr config.Conf, opts *fsOnlyContext) error {
-	for _, chg := range tr.Changes() {
-		if chg == "core.system.network.netplan" || strings.HasPrefix(chg, "core.system.network.netplan.") {
-			return fmt.Errorf("cannot set netplan config yet")
+var snapstateStore = func(st *state.State, deviceCtx snapstate.DeviceContext) connectivityCheckStore {
+	return snapstate.Store(st, deviceCtx)
+}
+
+func storeReachable(st *state.State) error {
+	st.Lock()
+	sto := snapstateStore(st, nil)
+	st.Unlock()
+	status, err := sto.ConnectivityCheck()
+	if err != nil {
+		return err
+	}
+
+	var unreachable []string
```

Comment:
This is a short function so this doesn't matter at all here, but I'd call this `unreachableHosts`. In case you are wondering where I got this bad habit of commenting on function and variable names, it's from this one: https://www.youtube.com/watch?v=MBRoCdtZOYg :-)